test/python/mock/test_new_qasm_backend.py

```diff
@@ -0,0 +1,36 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test of qasm fake backends from qiskit.mock package."""
+import unittest
+from qiskit import QuantumCircuit, transpile
+from qiskit.test import QiskitTestCase
+from qiskit.providers.fake_provider import FakeBogota
+from qiskit.utils.backend_utils import has_aer
+
+
+class FakeQasmBackendsTest(QiskitTestCase):
+    """Tests for FakeQasmBackend"""
+
+    @unittest.skipUnless(has_aer(), "qiskit-aer is required to run this test")
+    def test_fake_qasm_backend_configured(self):
+        """Fake backends honor kwargs passed."""
+        backend = FakeBogota()  # this is a FakePulseBackend implementation
+
+        qc = QuantumCircuit(2)
+        qc.x(range(0, 2))
+        qc.measure_all()
+
+        trans_qc = transpile(qc, backend)
+        raw_counts = backend.run(trans_qc, shots=1000).result().get_counts()
+
+        self.assertEqual(sum(raw_counts.values()), 1000)
```

Comment:
What are the two new tests for?

Reply:
The tests are for the new imports. I didn't change old code as we need it still tested. Related to the comment above, if we replace all the usages right now, then the old code doesn't have tests anymore. I can do that, but maybe it is better to change all usages when the deprecation is introduced. If a new task is created, you can assign it to me.

Reply:
That's fair enough.  Really, the same code is tested as before, but this does test the new import paths, which is useful.